scripts/test-publish.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
💯  Awesome change! Really really good that we can test publishing without populating npm!